docs/src/FAQ.md

```diff
@@ -83,6 +83,8 @@ The list here is incomplete, but is vetted for quality.
 
  - [Automatic Differentiation for Dummies keynote video](https://www.youtube.com/watch?v=FtnkqIsfNQc) by [Simon Peyton Jones](https://github.com/simonpj): particularly good if you like pure math type thinking.
 
+ - ["What types work with differentiation?](https://github.com/google-research/dex-lang/issues/454#issuecomment-766089519) comment on DexLang GitHub issue by [Dan Zhgeng](https://github.com/dan-zheng): summarizes several years of insights from the Swift AD work.
```

Comment:
Small name tweak if you don't mind!
```suggestion
 - ["What types work with differentiation?](https://github.com/google-research/dex-lang/issues/454#issuecomment-766089519) comment on DexLang GitHub issue by [Dan Zheng](https://github.com/dan-zheng): summarizes several years of insights from the Swift AD work.
```

Reply:
ah my inability to operate keyboard strikes again